transaction_integration_test.go

```diff
@@ -757,6 +757,9 @@ func TestTransactionSkipAdvancedFraudChecks(t *testing.T) {
 		},
 		Options: &TransactionOptions{
 			SkipAdvancedFraudChecking: true,
+			ThreeDSecure: &TransactionOptionsThreeDSecure{
+				Required: false,
+			},
```

Comment:
Can you move this into a new test, that tests specifically 3DS fields independent of the other risk data fields?